content/storage-history/history.textile

```diff
@@ -0,0 +1,596 @@
+    ```
+
+    ```[php]
+    $rest = new Ably\AblyRest('{{API_KEY}}');
+    $channel = $rest->channels->get('{{RANDOM_CHANNEL_NAME}}');
+    $channel->publish('example', 'message data');
+    $resultPage = $channel->history();
+    $recentMessage = $resultPage->items[0];
+    echo("Most recent message data: " . $recentMessage->data);
+    ```
+
+    ```[java]
+    AblyRest rest = new AblyRest("{{API_KEY}}");
+    Channel channel = rest.channels.get("{{RANDOM_CHANNEL_NAME}}");
+    channel.publish("example", "message data")
+    PaginatedResult<Message> resultPage = channel.history(null);
+    Message recentMessage = resultPage.items[0];
+    System.out.println("Most recent message: " + recentMessage.id + " - " + recentMessage.data);
+    ```
+
+    ```[csharp]
+    AblyRest rest = new AblyRest("{{API_KEY}}");
+    IRestChannel channel = rest.Channels.Get("{{RANDOM_CHANNEL_NAME}}");
+    await channel.PublishAsync("example", "message data");
+    PaginatedResult<Message> resultPage = await channel.HistoryAsync();
+    Message recentMessage = resultPage.Items[0];
+    Console.WriteLine("Most recent message: " + recentMessage.Id + " - " + recentMessage.Data);
+    ```
+
+    ```[objc]
+    ARTRest *rest = [[ARTRest alloc] initWithKey:@"{{API_KEY}}"];
+    ARTRestChannel *channel = [rest.channels get:@"{{RANDOM_CHANNEL_NAME}}"];
+    [channel publish:@"example" data:@"message data"];
+    [channel history:^(ARTPaginatedResult<ARTMessage *> *resultPage, ARTErrorInfo *error) {
+        ARTMessage *recentMessage = resultPage.items[0];
+        NSLog(@"Most recent message: %@ - %@", recentMessage.id, recentMessage.data);
+    }];
+    ```
+
+    ```[swift]
+    let rest = ARTRest(key: "{{API_KEY}}")
+    let channel = rest.channels.get("{{RANDOM_CHANNEL_NAME}}")
+    channel.publish("example", data: "message data")
+    channel.history { resultPage, error in
+        let recentMessage = resultPage!.items[0] as! ARTMessage
+        print("Most recent message: \(recentMessage.id) - \(recentMessage.data)")
+    }
+    ```
+
+    ```[go]
+    rest, err := ably.NewRestClient(ably.NewClientOptions("{{API_KEY}}"))
+    channel := rest.Channels.Get("{{RANDOM_CHANNEL_NAME}}")
+    err = channel.Publish("example", "message data")
+    page, err := channel.History(nil)
+    fmt.Println("Last published message: %s\n", page.Messages[0].Data)
+
+
+<span lang="ruby">Note that all examples on this page assume you are running them within an EventMachine reactor. Find out more in our "realtime usage documentation":/LINK-Me.</span>
+
+h4(#channel-parameters). Channel history parameters
+
+The following query parameters can be included in a call to @history()@. Note that @untilAttach@ is only available when using the realtime interface of an Ably SDK:
+
+|_. Parameter |_. Description |
+| options | ensures that clients disconnected abruptly from Ably can resume their connection |
```

Comment:
I'm not sure where this description came from. Options is the object containing the query parameters. Would it work better in the description above the table?